libsel4vmmplatsupport/src/fdt_utils.c

```diff
@@ -0,0 +1,27 @@
+/*
```

Comment:
I wonder if https://github.com/seL4/util_libs/tree/master/libfdt could be a better place for this. This is generic DTB handling utility code, that would be used outside of sel4vmm also.

Reply:
Sure it is a better place, but seems that it is an almost vanilla copy of `libfdt`. I was kind of pushed back because I thought it is that way in order to facilitate easier upgrading. Just now I realized there is actually that SPDX commit that makes it not vanilla anymore. I will create a PR in that repo too, so we can discuss which one is correct.